src/providers/krb5/krb5_child.c

```diff
@@ -4108,7 +4108,7 @@ int main(int argc, const char *argv[])
          _("Requests canonicalization of the principal name"), NULL},
         {CHILD_OPT_SSS_CREDS_PASSWORD, 0, POPT_ARG_NONE, &sss_creds_password,
          0, _("Use custom version of krb5_get_init_creds_password"), NULL},
-        {CHILD_OPT_CHAIN_ID, 0, POPT_ARG_LONG, &chain_id,
+        {"chain-id", 0, POPT_ARG_LONG, &chain_id,
```

Comment:
Why? What's wrong with the macro?

Reply:
I've removed macro from 'krb5_auth.h' because krb5 provider code shouldn't touch it anymore. So I could either add this macro to 'krb5_child.c" or use "chain-id" directly. I opted for the latter for the sake of consistency.

In general I'd like to add "chain-id" to 'SSSD_DEBUG_OPTS' (as well as dumpable" and "backtrace"), but this PR was already large enough and I didn't want to side track it.